.github/workflows/lint.yml

```diff
@@ -0,0 +1,38 @@
+name: Lint
+on:
+  push:
+    branches:
+      - 3.2.0_downgraded_deps
+      - master
+  pull_request:
+    branches:
+      - 3.2.0_downgraded_deps
+      - master
+  schedule:
+    - cron: '28 20 2 * *'
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    permissions:
+      checks: write
```

Comment:
Looks like this isn't working due to some nuance with forked repos, and that is causing the failure:

```
GITHUB_TOKEN Permissions
  Checks: read
  Metadata: read
```

Additionally, since there is no permissions section on the test workflow, test workflow has

```
GITHUB_TOKEN Permissions
  Actions: read
  Checks: read
  Contents: read
  Deployments: read
  Discussions: read
  Issues: read
  Metadata: read
  Packages: read
  Pages: read
  PullRequests: read
  RepositoryProjects: read
  SecurityEvents: read
  Statuses: read
```

Not harmful though since it's all `read` and a public repo.